pkg/encryption/cipher.go

```diff
@@ -126,3 +126,30 @@ func (c *Cipher) Decrypt(s string) (string, error) {
 
 	return string(encrypted), nil
 }
+
+// EncryptInto encrypts the value and stores it back in the string pointer
+func (c *Cipher) EncryptInto(s *string) error {
+	return into(c.Encrypt, s)
```

Comment:
@JoelSpeed In my size reduction analysis in #523 -- this technique was the biggest culprit of the bloat. `Encrypt` having an embedded base64 encoding so the result was a valid string that could be in JSON causes 33% size growth.

This is why I experimented shifting to MessagePack (since it supports internal binary data) and relatedly, moving the encryption after the fact to the whole payload, rather than encrypting inside the fields (for speed and to save a few bytes of AES-CFB IV headers present for each distinct inner encryption, especially for smaller non token fields).

Reply:
Ack. This PR isn't aimed at improving the size of the sessions at all but is more aimed at improving the code quality. I imagine what will happen is a lot of this may be undone once  we implement the MessagePack route, but for now, it shows progression towards better code and eventually the better session sizes